stylesheets/blue/components/_button_primary.scss

```diff
@@ -4,6 +4,7 @@
 //
 // default - Normal state
 // .ButtonPrimary--fullWidth - A button that takes the whole width of its parent
+// .ButtonPrimary--brightTurquoise - A button that uses the brightTurquoise color
```

Comment:
New insight: our brand color is blue and with the redesign there isn't much blue around, so the decision was to use blue buttons to maintain a bluish site. What do you think of changing the modifier to `ButtonPrimary--brand`?